doc/source/security/encrypt-traffic.rst

```diff
@@ -61,7 +61,7 @@ Traffic of each type is transferred via different channel, and so it is
 important to configure secure channels for all 3 variants to completely
 secure the replication traffic.
```

Comment:
This document should talk about this being a default setting in PXC 8.0 and we should explicitly mention that the default has changed and how to set up SSL or link to instructions.